filer/admin/imageadmin.py

```diff
@@ -90,11 +90,17 @@ class Media(object):
 
 
 class ImageAdmin(FileAdmin):
+    change_form_template = 'admin/filer/image/change_form.html'
     form = ImageAdminForm
 
 
+if FILER_IMAGE_MODEL == 'filer.Image':
+    extra_main_fields = ('author', 'default_alt_text', 'default_caption',),
+else:
+    extra_main_fields = ('default_alt_text', 'default_caption',),
```

Comment:
shouldn't we do a try/except to check if author field is present, instead of hardcoding check for image model?

Reply:
Here we really want to check if we are using the "original" filer model; In the end it doesn't really matter: if I extend the image model I will redefine the admin anyway
But in case you want to check field instead, I suggest using `if hasattr` instead